tests/SimplesheetTest.php

```diff
@@ -185,9 +186,9 @@ public function getCsvSettings(): array
 
         $contents = file_get_contents(__DIR__.'/Data/Disks/Local/filename.csv');
 
-        $this->assertStringContains('sep=;', $contents);
-        $this->assertStringContains('A1;B1', $contents);
-        $this->assertStringContains('A2;"B2 Test"', $contents);
+        $this->assertStringContainsString('sep=;', $contents);
+        $this->assertStringContainsString('A1;B1', $contents);
+        $this->assertStringContainsString('A2;"B2 Test"', $contents);
```

Comment:
Extending from `Nikazooz\Simplesheet\Tests\TestCase` allowed using existing compatibility fix for different PHPUnit versions. There's no real need to extend from `Orchestra\Testbench\TestCase` directly.